### PR TITLE
Add some docs info to the internal PR cover letter.

### PR DESCRIPTION
**Pull request by peter-fogg**
@nedbat @lamagnifica, I've added a bit of text for documentation in the base cover letter. How's this look?

### CODE REVIEW

**Comment by edx-webhook** [comment]
# PR Cover Letter
- [ ] A detailed description of the changes in the body of the PR
- [ ] If available, a link to the relevant JIRA ticket in the body of the PR
  - `[PROJ-XXXX](https://openedx.atlassian.net/browse/PROJ-XXXX)`

# Sandbox
- [ ] Build a sandbox for your branch and add a link here

# Testing
- [ ] i18n
- [ ] RTL
- [ ] Unit, integration, acceptance tests as appropriate
- [ ] Analytics
- [ ] Performance
- [ ] Database migrations are backwards-compatible

# Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [ ] Code review #1
- [ ] Code review #2
- [ ] UI strings/error msgs review
- [ ] UX review
- [ ] Accessibility review
- [ ] Product review

# DevOps assistance
If it's needed, or even if you suspect it might be needed, do any/all these things as early as possible. Don't surprise devops right before merging! For example, touching envs/aws.py or or envs/common.py are dead giveaways that you'll need assistance when pushing your changes out. Things that require devops attention include, but are not limited to:
- configuration changes
- feature flags
- migrations
Ways to get devops attention, pick-and-choose whichever is most appropriate for your specific PR:
- [ ] Tag devops on your PR (`@edx/devops`), either for review (more specific to devops = better) or just a general FYI
- [ ] File a support ticket

FYI: Tag anyone who might be interested in this PR here.

# Post-review
- [ ] Squash commits

File: openedx_webhooks/templates/github_pr_cover_letter.md.j2 [hunk 1]
```diff
@@ -18,11 +18,15 @@
...
 - [ ] Code review #2
-- [ ] UI strings/error msgs review
+- [ ] Docs review
 - [ ] UX review
 - [ ] Accessibility review
 - [ ] Product review
 
+# Documentation considerations
+- [ ] Are there user visible changes? If so, list who is affected, and include a pointer to the relevant code.
+- [ ] Should this be indicated in release notes? If so, include a draft description of your changes.
```
Comment (nedbat) [inline openedx_webhooks/templates/github_pr_cover_letter.md.j2]: "indicated" is odd here, how about "mentioned"?

File: openedx_webhooks/templates/github_pr_cover_letter.md.j2 [hunk 2]
```diff
@@ -18,11 +18,15 @@
 If you've been tagged for review, please check your corresponding box once you've given the :+1:.
 - [ ] Code review #1
 - [ ] Code review #2
-- [ ] UI strings/error msgs review
+- [ ] Docs review
```
Comment (lamagnifica) [inline openedx_webhooks/templates/github_pr_cover_letter.md.j2]: maybe Docs review (required for UI strings/error msgs)

File: openedx_webhooks/templates/github_pr_cover_letter.md.j2 (hunk 1 above)
Comment (lamagnifica) [inline openedx_webhooks/templates/github_pr_cover_letter.md.j2]: I also don't want people to think they need to repeat work, so maybe replace "If so, include a draft description of your changes." with "If so, make sure that you included a description"

**Comment by lamagnifica** [comment]
a couple small suggestions then :+1: . Thanks so much @peter-fogg!

**Comment by peter-fogg** [comment]
@nedbat @lamagnifica I like those suggestions, changes have been made.

**Pull request merged**